config.template.py

```diff
@@ -60,6 +60,15 @@ class Config:
     RUNJOB_TIMEOUT = 60
     COPYOUT_TIMEOUT = 30
 
+    # Docker constants
+    BOOT2DOCKER_INIT_TIMEOUT = 5
+    BOOT2DOCKER_START_TIMEOUT = 30
+    BOOT2DOCKER_ENV_TIMEOUT = 5
+    DOCKER_IMAGE_BUILD_TIMEOUT = 300
+    DOCKER_RM_TIMEOUT = 5
+    DOCKER_IMAGE = ''
+    DOCKER_VOLUME_PATH = ''
```

Comment:
Our general strategy with config should be things working out of the box. So I think we should ideally give a default value for these (as well as the courselabs directory) and put a setup.sh that will create the directories. If people would like to change the default values, they're free to do so.